doc/BuildProgramVisualStudio.md

```diff
@@ -0,0 +1,66 @@
+# Introduction #
+
+This is a first step in documenting what steps are needed to compile FreeRCT on Windows using Visual Studio. We hope to reduce the length of this list later on.
+
+# Details #
+
+We here assume F:\trunk is the path to a checkout/clone of trunk, and that you have a folder F:\libraries. You can replace these with actual paths on your system.
+
+This have been tested to build win32/x86 version. It may also work for x64, but this have not been tested.
+
+## Prepare: Install software ##
+  * Install CMake
+  * Install Visual Studio 2013 (It may work with earlier versions, but that is untested)
+  * Install an SVN client. Eg. TortoiseSVN
+
+## Prepare: Download files ##
+  1. Clone/check out FreeRCT source code from SVN to F:\trunk. There should not be a trunk folder inside F:\trunk. Instead you should have F:\trunk\src etc.
+  1. Download SDL2 development packet and unpack to F:\libraries\SDL2 https://www.libsdl.org/download-2.0.php
+  1. Download SDL2\_ttf development packet and unpack to F:\libraries\SDL2\_ttf https://www.libsdl.org/projects/SDL_ttf/
+  1. Download OpenTTD useful 5.1 ("Headers/libraries to compile with MSVC for windows") and unpack to F:\libraries\OpenTTD\_essentials\ http://www.openttd.org/en/download-openttd-useful
+  1. Download SDL2 and SDL2\_ttf runtime binaries from the links above and put the DLLs in F:\trunk\bin (you may need to create the bin folder). If you plan to distribute the bin directory, mind to include any license/copyright files from SDL and SDL\_ttf that you are required to include.
+
+## Prepare: Font config ##
+Get a font. Here we use Bitstream Vera that is an open font.
+  1. Go to http://www.dafont.com/bitstream-vera-sans.font and click on Download-button. Extract vera.ttf and put in F:\trunk\bin
+  1. Create F:\trunk\bin\freerct.cfg in your favoruite text editor:
+```
+[font]
+medium-size = 12
+medium-path = ./vera.ttf
```

Comment:
Windows can cope with / path separators

I presume this all works anyway, as it's just copied from what Zuu originally wrote